Source/WebCore/platform/SharedStringHash.h

```diff
@@ -31,18 +31,32 @@
 
 namespace WebCore {
 
-using SharedStringHash = uint32_t;
+struct SharedStringHash {
```

Comment:
It is a bit unfortunate that our previous IPC serialization format didn't require us to write a whole new struct but the new one does. I wish we'd fix this some other way (e.g. by adding some type of typedef support to the new serialization format).

The only thing we really want to do is use SharedStringHash in the serialization.in and make sure it does get serialized as an uint32_t. I don't think the approach in this patch is the best way to fix this. It is a lot of boilerplate code for no great reason.